content/ch-states/single-qubit-gates.ipynb

```diff
@@ -319,7 +319,7 @@
     "\n",
     "$$ |\\circlearrowleft\\rangle, \\quad |\\circlearrowright\\rangle$$\n",
     "\n",
-    "We leave it as an exercise to calculate these. There are in fact an infinite number of bases; to form one, we simply need two orthogonal vectors.\n",
+    "We leave it as an exercise to calculate these. There are in fact an infinite number of bases; to form one, we simply need two orthogonal vectors. Recall, from the appendix on linear algebra, that the eigenvectors of both Hermitian and unitary matrices form a basis for the vector space. Due to this property, we can be sure that the eigenstates of the X-gate and the Y-gate indeed form a basis for 1-qubit states.\n",
```

Comment:
```suggestion
    "We leave it as an exercise to calculate these. There are in fact an infinite number of bases; to form one, we simply need two orthogonal vectors. The eigenvectors of both Hermitian and unitary matrices form a basis for the vector space. Due to this property, we can be sure that the eigenstates of the X-gate and the Y-gate indeed form a basis for 1-qubit states (read more about this in the [linear algebra page](https://qiskit.org/textbook/ch-appendix/linear_algebra.html#Matrices-and-Matrix-Operations) in the appendix)\n",
```

We don't require people read the LA page beforehand so I think better to include it as a reference.